package.json

```diff
@@ -129,5 +129,10 @@
             "slate-hyperscript",
             "slate-react"
         ]
+    },
+    "pnpm": {
+        "overrides": {
+            "glob-parent@<5.1.2": ">=5.1.2"
```

Comment:
The reason for this override is because of the `gulp` package, which doesn't have a newer version. So, until we replace the `gulp` build pipeline with something else, we have to use this override.